src/logic/calculate.test.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Calculator from '../components/Calculator';
+
+it('Rendering 3', () => {
+  const tree = renderer.create(<Calculator />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] As mentioned in the project requirements all the `functionalities` & `scenarios` of the `calculate.js` modules should be covered using Jest, please kindly add them 🙏🏾